### PR TITLE
Portuguese translation of base template

### PR DESCRIPTION
**Pull request by HakaCode**
My first PR in this repo!
I hope I did everything correctly.

Makes progress on #505

### CODE REVIEW

File: src/templates/pt/2019/base.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web
+com os dados e as tendências do arquivo HTTP.{% endblock %}
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:4]: Combine these to keep line numbering same as English:

```suggestion
{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do arquivo HTTP.{% endblock %}
```

File: src/templates/pt/2019/base.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  <span class="pre">Por</span>
+  <span class="ha">HTTP Archive</span>
+</span>
+{% endblock %}
+
+{% block mission %}
+Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O
+Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web.
+É composto por 20 capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e
+distribuição.
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:26]: Combine these to keep line numbering same as English:

```suggestion
Nossa missão é combinar as estatísticas brutas e as tendências do HTTP Archive com a experiência da comunidade da web. O Web Almanac é um relatório abrangente sobre o estado da Web, apoiado por dados reais e especialistas confiáveis da Web. É composto por 20 capítulos que abrangem aspectos do conteúdo da página, experiência do usuário, publicação e distribuição.
```

File: src/templates/pt/2019/base.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} no Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} no GitHub{% endmacro %}
+{% macro website(authorName) %}{{authorName}} website{% endmacro %}
+
+{% macro edition(year) %}{{ year }} Edição{% endmacro %}
+
+{% block footer_title %}Navegação no rodapé{% endblock %}
+{% block copyright %}© Web Almanac. Licenciado sob <a
+  href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:83]: Combine these to keep line numbering same as English:

```suggestion
{% block copyright %}© Web Almanac. Licenciado sob <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}

```

File: src/templates/pt/2019/base.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+{% block accessibility_statement %}Declaração de acessibilidade{% endblock %}
+
+{% block contributors_description %}
+O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas
+ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:88]: Combine these to keep line numbering same as English:

```suggestion
O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
```

File: src/templates/pt/2019/base.html [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+{% block methodology_stat_1_title %}Sites testados{% endblock %}
+{% block methodology_stat_1 %}5.8M{% endblock %}
+{% block methodology_stat_2_title %}Dados processados{% endblock %}
+{% block methodology_stat_2 %}20.9 TB{% endblock %}
+{% block methodology_description %}
+Salvo indicação em contrário, as métricas em todos os 20 capítulos do Web Almanac são originárias do conjunto de dados
+do HTTP Archive. O HTTP Archive é um projeto administrado pela comunidade que acompanha como a Web é criada desde 2010.
+Usando o WebPageTest e o Lighthouse, os metadados de cerca de 6 milhões de sites são testados mensalmente e incluídos em
+um banco de dados público do BigQuery para análise. O conjunto de dados de julho de 2019 foi usado como base para as
+métricas do Web Almanac. Para mais informações, consulte a página Metodologia.
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:101]: Combine these to keep line numbering same as English:

```suggestion
Salvo indicação em contrário, as métricas em todos os 20 capítulos do Web Almanac são originárias do conjunto de dados do HTTP Archive. O HTTP Archive é um projeto administrado pela comunidade que acompanha como a Web é criada desde 2010. Usando o WebPageTest e o Lighthouse, os metadados de cerca de 6 milhões de sites são testados mensalmente e incluídos em um banco de dados público do BigQuery para análise. O conjunto de dados de julho de 2019 foi usado como base para as métricas do Web Almanac. Para mais informações, consulte a página Metodologia.
```

File: src/templates/pt/2019/base.html [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+{% block foreword %}
+<p>
+  A web aberta é uma rede de tecnologias incrivelmente complexa e em evolução. Indústrias e carreiras inteiras são
+  construídas na Web e dependem de seu vibrante ecossistema para ter sucesso. Por mais crítica que seja a web, entender
+  como está se saindo tem sido surpreendentemente evasivo. Desde 2010, a missão do projeto HTTP Archive é rastrear como
+  a Web é construída e está fazendo um trabalho incrível. No entanto, houve uma lacuna que foi especialmente difícil de
+  resolver: trazer significado aos dados que o projeto HTTP Archive está coletando e permitir que a comunidade entenda
+  facilmente o desempenho da web. É aí que entra o Web Almanac.
+</p>
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:117]: Combine these to keep line numbering same as English:

```suggestion
    <p>
      A web aberta é uma rede de tecnologias incrivelmente complexa e em evolução. Indústrias e carreiras inteiras são construídas na Web e dependem de seu vibrante ecossistema para ter sucesso. Por mais crítica que seja a web, entender como está se saindo tem sido surpreendentemente evasivo. Desde 2010, a missão do projeto HTTP Archive é rastrear como a Web é construída e está fazendo um trabalho incrível. No entanto, houve uma lacuna que foi especialmente difícil de resolver: trazer significado aos dados que o projeto HTTP Archive está coletando e permitir que a comunidade entenda facilmente o desempenho da web. É aí que entra o Web Almanac.
    </p>
```

File: src/templates/pt/2019/base.html [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+<p>
+  A missão do Web Almanac é levar o tesouro de insights que, de outra forma, seriam acessíveis apenas a intrépidos
+  mineradores de dados e empacotá-lo de uma maneira fácil de entender. Isso é possível com a ajuda de especialistas do
+  setor, que podem entender os dados e nos dizer o que eles significam. Cada um dos 20 capítulos do Web Almanac se
+  concentra em um aspecto específico da Web, e cada um foi criado e revisado por especialistas em seu campo. A força do
+  Web Almanac flui diretamente da experiência das pessoas que o escrevem.
+</p>
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:125]: Combine these to keep line numbering same as English:

```suggestion
    <p>
      A missão do Web Almanac é levar o tesouro de insights que, de outra forma, seriam acessíveis apenas a intrépidos mineradores de dados e empacotá-lo de uma maneira fácil de entender. Isso é possível com a ajuda de especialistas do setor, que podem entender os dados e nos dizer o que eles significam. Cada um dos 20 capítulos do Web Almanac se concentra em um aspecto específico da Web, e cada um foi criado e revisado por especialistas em seu campo. A força do Web Almanac flui diretamente da experiência das pessoas que o escrevem.
    </p>
```

File: src/templates/pt/2019/base.html [hunk 8]
```diff
@@ -0,0 +1,229 @@
...
+
+<p>
+  Muitas das conclusões do Web Almanac são dignas de comemoração, mas também é um lembrete importante do trabalho
+  ainda necessário para oferecer experiências de usuário de alta qualidade. As análises orientadas por dados em cada
+  capítulo são uma forma de responsabilidade que todos compartilhamos pelo desenvolvimento de uma Web melhor. Não se
+  trata de envergonhar aqueles que estão errando, mas de iluminar o caminho das melhores práticas, para que haja uma
+  maneira clara e correta de fazer as coisas. Com a ajuda contínua da comunidade da web, esperamos fazer disso uma
+  tradição anual, para que a cada ano possamos acompanhar nosso progresso e fazer as correções necessárias, conforme
+  necessário.
+</p>
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:135]: Combine these to keep line numbering same as English:

```suggestion
    <p>
      Muitas das conclusões do Web Almanac são dignas de comemoração, mas também é um lembrete importante do trabalho ainda necessário para oferecer experiências de usuário de alta qualidade. As análises orientadas por dados em cada capítulo são uma forma de responsabilidade que todos compartilhamos pelo desenvolvimento de uma Web melhor. Não se trata de envergonhar aqueles que estão errando, mas de iluminar o caminho das melhores práticas, para que haja uma maneira clara e correta de fazer as coisas. Com a ajuda contínua da comunidade da web, esperamos fazer disso uma tradição anual, para que a cada ano possamos acompanhar nosso progresso e fazer as correções necessárias, conforme necessário.
    </p>
```

File: src/templates/pt/2019/base.html [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+
+<p>
+  Há muito o que aprender neste relatório. Portanto, comece a explorar e compartilhar suas sugestões com a comunidade
+  para que possamos avançar coletivamente em nossa compreensão do estado da web.
+</p>
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:140]: Combine these to keep line numbering same as English:

```suggestion
    <p>
      Há muito o que aprender neste relatório. Portanto, comece a explorar e compartilhar suas sugestões com a comunidade para que possamos avançar coletivamente em nossa compreensão do estado da web.
    </p>
```

File: src/templates/pt/2019/base.html [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+
+<p>— <em><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}">Rick Viscomi</a>, criador do
+    Web Almanac</em></p>
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:143]: Combine these to keep line numbering same as English:

```suggestion
    <p>— <em><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}">Rick Viscomi</a>, criador do Web Almanac</em></p>
```

File: src/templates/pt/2019/base.html [hunk 11]
```diff
@@ -0,0 +1,229 @@
...
+  "developers": "Desenvolvedores",
+  "editors": "Editores",
+  "leads": "Lider",
+  "reviewers": "Revisores",
+  "translators": "Tradutores",
+  }
+%}
+
+{% set localizedContributors = {
+  "kleinab": 'Abigail Klein is a Google software engineer. She worked on Google Docs, Sheets, and Slides web accessibility where she added <a href="https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/">automatic captions to Google Slides</a>, as well as improving screen reader, braille, screen magnifier, and high contrast support. She currently works on Google Chrome and ChromeOS accessibility. She has a bachelor\'s and master\'s degree in computer science from MIT, where she co-founded an assistive technology hackathon and was a lab assistant and guest lecturer of the assistive technology class.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:200]: Can we translate bios these too?

File: src/templates/pt/2019/base.html [hunk 12]
```diff
@@ -0,0 +1,229 @@
...
+  "ymschaap": 'Founder at technical SEO consultancy <a href="https://build.amsterdam/">build.amsterdam</a>. Previously founded several web companies that reached over 1 billions users. Blogging about his latest (ad)ventures since 2005 on <a href="https://yvoschaap.com/">yvoschaap.com</a>.',
+  "zachleat": 'Zach is a Web Developer with <a href="https://www.filamentgroup.com/">Filament Group</a>. He’s currently fixated on <a href="https://www.zachleat.com/web/fonts/">web fonts</a> and <a href="https://www.zachleat.com/web/introducing-eleventy/">static site generators</a>. His <a href="https://www.zachleat.com/web/speaking/">public speaking résumé</a> includes talks in eight different countries at events like JAMstack_conf, Beyond Tellerrand, Smashing Conference, CSSConf, and <a href="https://www.zachleat.com/web/whitehouse/">The White House</a>. He also helps herd <a href="http://nejsconf.com/">NEJS CONF</a> and the <a href="http://nebraskajs.com">NebraskaJS</a> meetup.',
+  }
+%}
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:229]: Add a blank newline at end of all files:

```suggestion
%}

```

File: src/templates/pt/2019/base.html (hunk 1 above)
Comment (HakaCode) [reply src/templates/pt/2019/base.html:4]: strange that, for me it was line by line, I believe it is the beautify plugin.
Thanks for reminding me @bazzadp

File: src/templates/pt/2019/base.html (hunk 11 above)
Comment (HakaCode) [reply src/templates/pt/2019/base.html]: @bazzadp Sorry I'm late.

I made the suggested changes, could you guide me on how to send the new update please?

File: src/templates/pt/2019/base.html [hunk 13]
```diff
@@ -0,0 +1,203 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do arquivo HTTP.{% endblock %}
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:3]: Should we leave HTTP Archive as a product name [as per the Translation guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide)

```suggestion
{% block description %}Web Almanac é um estado anual do relatório da Web que combina a experiência da comunidade da Web com os dados e as tendências do HTTP Archive.{% endblock %}
```

File: src/templates/pt/2019/base.html [hunk 14]
```diff
@@ -0,0 +1,203 @@
...
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+
+{% block author %}Autor(a){% endblock %}
+{% block authors %}Autores{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} no Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} no GitHub{% endmacro %}
+{% macro website(authorName) %}{{authorName}} website{% endmacro %}
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:73]: We just added a new one!

```suggestion
{% macro onLinkedIn(authorName) %}{{authorName}} no LinkedIn{% endmacro %}
{% macro website(authorName) %}{{authorName}} website{% endmacro %}
```

File: src/templates/pt/2019/base.html [hunk 15]
```diff
@@ -0,0 +1,203 @@
...
+  "bkardell": 'Brian Kardell é desenvolvedor advogado no <a href="https://igalia.com">Igalia</a>, colaborador normas, <a href="https://bkardell.com">blogger</a>, e é atualmente o representante W3C Comité Consultivo para a <a href="https://openjsf.org/">Open JS Foundation</a>. Ele foi o fundador do Extensible Web Community Group e co-autor de <a href="https://extensiblewebmanifesto.org">The Extensible Web Manifesto</a>.',
+  "colinbendell": 'Colin é parte do escritório CTO da <a href="https://cloudinary.com/">Cloudinary</a>  co-autor do O \' Reilly livro <a href="https://www.oreilly.com/library/view/high-performance-images/9781491925799/">High Performance Images</a>. Ele passa grande parte do tempo na interseção de dados, mídias, navegadores e padrões de alto volume. Você pode encontrá-lo no Twitter <a href="https://twitter.com/colinbendell">@colinbendell</a> e em blogs em <a href="https://bendell.ca/">https://bendell.ca</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:182]:
```suggestion
  "colinbendell": 'Colin é parte do escritório CTO da <a href="https://cloudinary.com/">Cloudinary</a>  co-autor do O\'Reilly livro <a href="https://www.oreilly.com/library/view/high-performance-images/9781491925799/"><i lang="en">High Performance Images</i></a>. Ele passa grande parte do tempo na interseção de dados, mídias, navegadores e padrões de alto volume. Você pode encontrá-lo no Twitter <a href="https://twitter.com/colinbendell">@colinbendell</a> e em blogs em <a href="https://bendell.ca/">https://bendell.ca</a>.',
```

File: src/templates/pt/2019/base.html [hunk 16]
```diff
@@ -0,0 +1,203 @@
...
+  "patrickhulce": 'Patrick Hulce é um ex-engenheiro Chrome, fundador do <a href="https://eris.ventures/">Eris Ventures</a>, membro da equipe do núcleo de <a href="https://github.com/GoogleChrome/lighthouse">Lighthouse</a> e <a href="https://github.com/GoogleChrome/lighthouse-ci">Lighthouse CI</a>, co-organizador do <a href="https://www.meetup.com/DallasJS/">DallasJS</a> meetup, e autor do projeto <a href="https://github.com/patrickhulce/third-party-web">third-party-web</a>.',
+  "paulcalvano": 'Paul Calvano é um Arquiteto de Desempenho Web em <a href="https://www.akamai.com/">Akamai</a>, onde ele ajuda as empresas a melhorar o desempenho de seus sites. Ele também é co-mantenedor do projeto HTTP Archive. Você pode encontrá-lo twittar em <a href="https://twitter.com/paulcalvano">@paulcalvano</a>, o blog em <a href="https://paulcalvano.com">http://paulcalvano.com</a> E compartilhamento de pesquisa HTTP Arquivo em <a href="https://discuss.httparchive.org">https://discuss.httparchive.org</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:191]:
```suggestion
  "paulcalvano": 'Paul Calvano é um Arquiteto de Desempenho Web em <a href="https://www.akamai.com/">Akamai</a>, onde ele ajuda as empresas a melhorar o desempenho de seus sites. Ele também é co-mantenedor do projeto HTTP Archive. Você pode encontrá-lo twittar em <a href="https://twitter.com/paulcalvano">@paulcalvano</a>, o blog em <a href="https://paulcalvano.com">http://paulcalvano.com</a> E compartilhamento de pesquisa HTTP Archive em <a href="https://discuss.httparchive.org">https://discuss.httparchive.org</a>.',
```

File: src/templates/pt/2019/base.html [hunk 17]
```diff
@@ -0,0 +1,203 @@
...
+  "ScottHelme": 'Scott Helme é um pesquisador de segurança e fundador da <a href="https://report-uri.com">report-uri.com</a> e <a href="https://securityheaders.com">securityheaders.com</a>. Você pode encontrá-lo falar sobre segurança no Twitter <a href="https://twitter.com/Scott_Helme">@Scott_Helme</a> e bloga em <a href="https://scotthelme.co.uk">scotthelme.co.uk</a>.',
+  "tammyeverts": 'Tammy Everts passou mais de duas décadas estudando usabilidade e UX. Nos últimos dez anos, ela se concentrou na interseção do UX com desempenho e negócios da Web. Ela é CXO em <a href="https://speedcurve.com/">SpeedCurve</a>, co-presidente do <a href="https://perfnow.nl/">performance.now() conference</a>, e autor do livro O\ 'Reilly <em><a href="http://shop.oreilly.com/product/0636920041450.do">Time Is Money: The Business Value of Performance</a></em>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:197]:
```suggestion
  "tammyeverts": 'Tammy Everts passou mais de duas décadas estudando usabilidade e UX. Nos últimos dez anos, ela se concentrou na interseção do UX com desempenho e negócios da Web. Ela é CXO em <a href="https://speedcurve.com/">SpeedCurve</a>, co-presidente do <a href="https://perfnow.nl/">performance.now() conference</a>, e autor do livro O\'Reilly <em><a href="http://shop.oreilly.com/product/0636920041450.do"><i lang="en">Time Is Money: The Business Value of Performance</i></a></em>.',
```

File: src/templates/pt/2019/base.html [hunk 18]
```diff
@@ -0,0 +1,203 @@
...
+  "amedina": 'Alberto Medina é advogado do desenvolvedor da equipe de ecossistemas de conteúdo da Web do Google, com foco no avanço da proliferação de conteúdo de qualidade na web por meio de tecnologias progressivas como Amp e o uso de APIs da Web modernas. Atualmente, o trabalho de Alberto tem um forte foco em sistemas de gerenciamento de conteúdo, pois ele lidera uma área do ecossistema de conteúdo chamada CMS Developer Relations.',
+  "andydavies": 'Andy Davies é consultor freelancer de desempenho na Web e ajudou alguns dos principais varejistas, jornais e empresas de serviços financeiros do Reino Unido a tornar seus sites mais rápidos. Ele escreveu o Pocket Guide to Web Performance, é co-autor do Using WebPageTest e também organizador do encontro sobre o London Web Performance. Você pode encontrar Andy no Twitter como <a href="https://twitter.com/andydavies">@AndyDavies</a>, e ele ocasionalmente bloga em <a href="https://andydavies.me">https://andydavies.me</a>',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:178]:
```suggestion
  "andydavies": 'Andy Davies é consultor freelancer de desempenho na Web e ajudou alguns dos principais varejistas, jornais e empresas de serviços financeiros do Reino Unido a tornar seus sites mais rápidos. Ele escreveu o <i lang="en">Pocket Guide to Web Performance</i>, é co-autor do Using WebPageTest e também organizador do encontro sobre o London Web Performance. Você pode encontrar Andy no Twitter como <a href="https://twitter.com/andydavies">@AndyDavies</a>, e ele ocasionalmente bloga em <a href="https://andydavies.me">https://andydavies.me</a>',
```

File: src/templates/pt/2019/base.html [hunk 19]
```diff
@@ -0,0 +1,203 @@
...
+  "arturjanc": 'Artur Janc é um engenheiro de segurança da informação do Google, que trabalha no design e na adoção de mecanismos de segurança de plataformas da Web no Google e na Web em geral. Ele argumenta com as pessoas na internet como <a href="https://twitter.com/arturjanc">@arturjanc on Twitter</a>.',
+  "bazzadp": 'Barry Pollard é um desenvolvedor de software e autor do livro Manning <a href="https://www.manning.com/books/http2-in-action">HTTP/2 in Action</a>. Ele acha a web incrível, mas quer torná-la ainda melhor. Você pode encontrá-lo twittar <a href="https://twitter.com/tunetheweb">@tunetheweb</a> e bloga em <a href="https://www.tunetheweb.com">www.tunetheweb.com</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:180]:
```suggestion
  "bazzadp": 'Barry Pollard é um desenvolvedor de software e autor do livro Manning <a href="https://www.manning.com/books/http2-in-action"><i lang="en">HTTP/2 in Action</i></a>. Ele acha a web incrível, mas quer torná-la ainda melhor. Você pode encontrá-lo twittar <a href="https://twitter.com/tunetheweb">@tunetheweb</a> e bloga em <a href="https://www.tunetheweb.com">www.tunetheweb.com</a>.',
```

File: src/templates/pt/2019/base.html [hunk 20]
```diff
@@ -0,0 +1,203 @@
...
+  "bkardell": 'Brian Kardell é desenvolvedor advogado no <a href="https://igalia.com">Igalia</a>, colaborador normas, <a href="https://bkardell.com">blogger</a>, e é atualmente o representante W3C Comité Consultivo para a <a href="https://openjsf.org/">Open JS Foundation</a>. Ele foi o fundador do Extensible Web Community Group e co-autor de <a href="https://extensiblewebmanifesto.org">The Extensible Web Manifesto</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:181]:
```suggestion
  "bkardell": 'Brian Kardell é desenvolvedor advogado no <a href="https://igalia.com">Igalia</a>, colaborador normas, <a href="https://bkardell.com">blogger</a>, e é atualmente o representante W3C Comité Consultivo para a <a href="https://openjsf.org/"><i lang="en">Open JS Foundation</i></a>. Ele foi o fundador do <i lang="en">Extensible Web Community Group</i> e co-autor de <a href="https://extensiblewebmanifesto.org"><i lang="en">The Extensible Web Manifesto</i></a>.',
```

File: src/templates/pt/2019/base.html [hunk 21]
```diff
@@ -0,0 +1,203 @@
...
+  "paulcalvano": 'Paul Calvano é um Arquiteto de Desempenho Web em <a href="https://www.akamai.com/">Akamai</a>, onde ele ajuda as empresas a melhorar o desempenho de seus sites. Ele também é co-mantenedor do projeto HTTP Archive. Você pode encontrá-lo twittar em <a href="https://twitter.com/paulcalvano">@paulcalvano</a>, o blog em <a href="https://paulcalvano.com">http://paulcalvano.com</a> E compartilhamento de pesquisa HTTP Arquivo em <a href="https://discuss.httparchive.org">https://discuss.httparchive.org</a>.',
+  "rachellcostello": 'Rachel Costello é uma SEO técnica &amp; Content Manager em <a href="https://www.deepcrawl.com/">DeepCrawl</a> um conferencista internacional que gasta seu tempo pesquisando e comunicar os mais recentes desenvolvimentos em pesquisa e. Rachel atualmente administra a produção de <a href="https://www.deepcrawl.com/knowledge/white-papers/">technical SEO white papers</a> e peças de pesquisa para DeepCrawl, e é um colunista regular para <a href="https://www.searchenginejournal.com/author/rachel-costello/">Search Engine Journal</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:192]: Should we translate technical SEO white papers? Not sure if there is an equivalent phrase in Portuguese and if you use the term "white paper"?

File: src/templates/pt/2019/base.html [hunk 22]
```diff
@@ -0,0 +1,203 @@
...
+  "ernee": 'Renee Johnson é consultora de produtos e web, entusiasta do WordPress, organizadora e voluntária frequente do WordCamp. Atualmente, ela trabalha com a equipe de Relações com desenvolvedores do sistema de gerenciamento de conteúdo do Google como especialista em suporte ao produto. ',
+  "rviscomi": 'Rick Viscomi é engenheiro sênior de programas de desenvolvedor do Google, trabalhando em projetos de transparência na Web como o HTTP Archive e o Chrome UX Report, e estudando a interseção de como os sites são construídos e experimentados. Rick é o anfitrião de <a href="https://www.youtube.com/playlist?list=PLNYkxOF6rcIBGvYSYO-VxOsaYQDw5rifJ">The State of the Web</a> em que especialistas discutem como a web está tendendo. Rick é co-autor de <a href="https://usingwpt.com">Using WebPageTest</a>, Um guia para testar o desempenho da web, e escreve com frequência sobre a web em <a href="https://dev.to/rick_viscomi">dev.to</a> e no Twitter em <a href="https://twitter.com/rick_viscomi">@rick_viscomi</a>.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:194]:
```suggestion
  "rviscomi": 'Rick Viscomi é engenheiro sênior de programas de desenvolvedor do Google, trabalhando em projetos de transparência na Web como o HTTP Archive e o Chrome UX Report, e estudando a interseção de como os sites são construídos e experimentados. Rick é o anfitrião de <a href="https://www.youtube.com/playlist?list=PLNYkxOF6rcIBGvYSYO-VxOsaYQDw5rifJ"><i lang="en">The State of the Web</i></a> em que especialistas discutem como a web está tendendo. Rick é co-autor de <a href="https://usingwpt.com"><i lang="en">Using WebPageTest</i></a>, Um guia para testar o desempenho da web, e escreve com frequência sobre a web em <a href="https://dev.to/rick_viscomi">dev.to</a> e no Twitter em <a href="https://twitter.com/rick_viscomi">@rick_viscomi</a>.',
```

File: src/templates/pt/2019/base.html [hunk 23]
```diff
@@ -0,0 +1,203 @@
...
+  "samdutton": 'Sam Dutton trabalha com a equipe do Google Chrome como advogado do desenvolvedor desde 2011. Ele organizou e apresentou em vários eventos, criou e ministrou vários cursos de desenvolvimento da web e trabalhou em vários vídeos, codelabs e orientação por escrito sobre PWA, desempenho, mídia, imagem e iniciativas de \'Next Billion Users\'. Ele mantém <a href="https://simpl.info">simpl.info</a>, que fornece mais simples exemplos possíveis de HTML, CSS e JavaScript. Sam cresceu no sul da Austrália, estudou em Sydney e vive desde 1986 em Londres.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:195]:
```suggestion
  "samdutton": 'Sam Dutton trabalha com a equipe do Google Chrome como advogado do desenvolvedor desde 2011. Ele organizou e apresentou em vários eventos, criou e ministrou vários cursos de desenvolvimento da web e trabalhou em vários vídeos, codelabs e orientação por escrito sobre PWA, desempenho, mídia, imagem e iniciativas de \'<i lang="en">Next Billion Users</i>\'. Ele mantém <a href="https://simpl.info">simpl.info</a>, que fornece mais simples exemplos possíveis de HTML, CSS e JavaScript. Sam cresceu no sul da Austrália, estudou em Sydney e vive desde 1986 em Londres.',
```

File: src/templates/pt/2019/base.html [hunk 24]
```diff
@@ -0,0 +1,203 @@
...
+  "tomayac": 'Thomas Steiner é um advogado de desenvolvedores da Web no Google Hamburg, focado em tornar a Web um lugar melhor por meio da padronização, criação e compartilhamento de melhores práticas e pesquisa. Ele bloga em <a href="https://blog.tomayac.com/">blog.tomayac.com</a> e os tweets como <a href="https://twitter.com/tomayac">@tomayac</a>.',
+  "una": 'Una Kravets é palestrante pública internacional do Brooklyn, redatora técnica e advogada de desenvolvedores de design de materiais no Google. A Una hospeda <a href="https://www.youtube.com/watch?v=YK8GZBx3hpg">Designing the Browser</a> e a série <a href="https://spec.fm/podcasts/toolsday">Toolsday</a> podcast de desenvolvedor . Siga-a no <a href="https://twitter.com/una">Twitter</a> para encontrar suas reflexões sobre CSS criativo, experiências do usuário e práticas recomendadas de desenvolvimento da web.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:199]:
```suggestion
  "una": 'Una Kravets é palestrante pública internacional do Brooklyn, redatora técnica e advogada de desenvolvedores de design de materiais no Google. A Una hospeda <a href="https://www.youtube.com/watch?v=YK8GZBx3hpg"><i lang="en">Designing the Browser</i></a> e a série <a href="https://spec.fm/podcasts/toolsday">Toolsday</a> podcast de desenvolvedor . Siga-a no <a href="https://twitter.com/una">Twitter</a> para encontrar suas reflexões sobre CSS criativo, experiências do usuário e práticas recomendadas de desenvolvimento da web.',
```

File: src/templates/pt/2019/base.html [hunk 25]
```diff
@@ -0,0 +1,203 @@
...
+  "ymschaap": 'Fundador técnico da consultoria SEO  <a href="https://build.amsterdam/">build.amsterdam</a>. Anteriormente, fundou várias empresas da web que atingiram mais de 1 bilhão de usuários. Blogar sobre suas últimas (ad) ventures desde 2005 em <a href="https://yvoschaap.com/">yvoschaap.com</a>.',
+  "zachleat": 'Zach é um desenvolvedor Web com <a href="https://www.filamentgroup.com/">Filament Group</a>. Ele está atualmente fixado em <a href="https://www.zachleat.com/web/fonts/">web fonts</a> e <a href="https://www.zachleat.com/web/introducing-eleventy/">static site generators</a>. Sua <a href="https://www.zachleat.com/web/speaking/">public speaking résumé</a> inclui palestras em oito países diferentes em eventos como JAMstack_conf, Beyond Tellerrand, Smashing Conference, CSSConf, e <a href="https://www.zachleat.com/web/whitehouse/">The White House</a>. Ele também ajuda em <a href="http://nejsconf.com/">NEJS CONF</a> e o <a href="http://nebraskajs.com">NebraskaJS</a> meetup.',
```
Comment (bazzadp) [inline src/templates/pt/2019/base.html:201]: Should we translate "static site generators"?

And meetup? Or is that a well-knownused phrase in Portuguese now?

File: src/templates/pt/2019/base.html (hunk 13 above)
Comment (HakaCode) [reply src/templates/pt/2019/base.html:3]: right, it was my fault in the last revision.

File: src/templates/pt/2019/base.html (hunk 25 above)
Comment (HakaCode) [reply src/templates/pt/2019/base.html:202]: I believe that some terms should not be translated, but if you want I can translate without problems.

File: src/templates/pt/2019/base.html (hunk 21 above)
Comment (HakaCode) [reply src/templates/pt/2019/base.html:193]: > Should we translate technical SEO white papers? Not sure if there is an equivalent phrase in Portuguese and if you use the term "white paper"?

I can't get a legal term to describe "technical SEO white papers", oh unless some other translator with better interpretation skills finds a more coherent term

**Comment by bazzadp** [comment]
Thanks for that. Looking forward to getting more of it translated! 🇧🇷

Btw noticed you are using the `main` branch of your fork. I wrote something up recently on why I think it’s better not to do that: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide#recommended-way-to-use-git-for-the-web-almanac

**Comment by HakaCode** [comment]
> Obrigado por isso. Ansioso para obter mais do que traduzido!🇧🇷
> 
> Btw percebeu que você está usando o `main`ramo do seu garfo. Escrevi recentemente algo sobre por que acho que é melhor não fazer isso: [https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide#recommended-way-to-use-git-for- o almanaque da web](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Git-Guide#recommended-way-to-use-git-for-the-web-almanac)

Thanks for accepting, in case I do something wrong in the future you can tell me, and also thanks for Patience with the newbie.

**Comment by bazzadp** [comment]
Wrong is the wrong word - this is very much a matter of opinion here. But have a read of that and see if you can understand why I suggest what I did. Happy to answer any questions!

**Pull request merged**